### PR TITLE
fix #72 Upgrade CRI to proto3 

### LINKED ISSUE #72: Upgrade CRI to proto3
CRI has changed its API to proto3. We should upgrade our vendor.

### PR DESCRIPTION
**Pull request by heartlock**
Update CRI in frakti to proto v3

Ref: https://github.com/kubernetes/kubernetes/pull/39158

### CODE REVIEW

**Comment by heartlock** [comment]
@feiskyer @resouer ready for review

File: pkg/hyper/hyper.go [hunk 1]
```diff
@@ -92,20 +92,20 @@ func (h *Runtime) Version() (string, string, string, error) {
 // Status returns the status of the runtime.
 func (h *Runtime) Status() (*kubeapi.RuntimeStatus, error) {
 	runtimeReady := &kubeapi.RuntimeCondition{
-		Type:   proto.String(kubeapi.RuntimeReady),
-		Status: proto.Bool(true),
+		Type:   *proto.String(kubeapi.RuntimeReady),
```
Comment (resouer) [inline pkg/hyper/hyper.go]: No need to use `*proto.String()`, just use `kubeapi.RuntimeReady` directly here.

Same with other places.

File: pkg/hyper/container.go [hunk 2]
```diff
@@ -156,15 +156,15 @@ func (h *Runtime) ListContainers(filter *kubeapi.ContainerFilter) ([]*kubeapi.Co
 		}
 
 		if filter != nil {
-			if filter.Id != nil && c.ContainerID != filter.GetId() {
+			if filter.Id != "" && c.ContainerID != filter.Id {
```
Comment (heartlock) [inline pkg/hyper/container.go]: it's necessary to check `filter.Id != ""` ,because `filter.Id == ""` means match all containerID in list

**Comment by resouer** [comment]
Thanks, generally LGTM. Leaving to @feiskyer to make final decision.

File: pkg/hyper/container.go (hunk 2 above)
Comment (resouer) [inline pkg/hyper/container.go]: ok, sgtm

**Comment by resouer** [comment]
Please update the godep with scripts and I will merge this.

**Comment by resouer** [comment]
I will merge this PR first, because PR 74 relies on this.

**Pull request merged**